etc/adminhtml/system.xml

```diff
@@ -984,6 +984,78 @@
                 </field>
             </group>
         </section>
+        <section id="algoliasearch_indexing_manager" translate="label" type="text" sortOrder="66" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Indexing Manager</label>
+            <tab>algolia</tab>
+            <resource>Algolia_AlgoliaSearch::algolia_algoliasearch</resource>
+            <group id="full_indexing" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                <label>Full indexing via Magento indexers</label>
+                <comment>Since version 3.16.0, the following Magento indexers are now disabled by default. We highly suggest to keep this configuration as it is and run the newly created CLI commands instead.</comment>
```

Comment:
I think we may need to clarify our language here that these configurations only disable "automatic full indexing" as we are not (or should not be) disabling indexers completely because we want to handle **delta** operations. We do mention "Full indexing via Magento indexers" in the title so that's good but reinforcing this idea further below would help eliminate potential confusion. 

Of course another option is to provide _even more granularity_ in the configuration in case they _want_ to disable delta indexers... 🤔  That would mean two sets of configurations... Thoughts?

Reply:
The goal of this section of the admin will be enhanced in the future during the different phases of the indexing improvements, and introducing more granularity can definitely be part of this enhancement :) .
In the meantime, I'll try to be more explicit.